buildSrc/subprojects/jvm/src/main/kotlin/gradlebuild.api-parameter-names-index.gradle.kts

```diff
@@ -36,6 +36,7 @@ val parameterNamesIndex by tasks.registering(ParameterNamesIndex::class) {
         project.layout.buildDirectory.file(moduleIdentity.baseName.map { "generated-resources/$it-parameter-names/$it-parameter-names.properties" })
     )
 }
+
```

Comment:
In case you wonder this is because the cache is busted on CI, this shouldn't be necessary at all.